charts/team-ns/templates/argocd/argocd-applicationset.yaml

```diff
@@ -57,6 +57,10 @@ spec:
           parameters:
             - name: sidecar.istio.io/inject
               value: 'true'
+            {{- if $v.apps.istio.defaultRevision }}
```

Comment:
Given async nature of argocd. Are you sure that these revision are going to be used during the istio upgrade?